src/main.rs

```diff
@@ -102,6 +105,16 @@ fn settings() -> io::Result<Settings> {
                     DEFAULT_THRESHOLD.code_str(),
                 )),
         )
+        .arg(
+            Arg::with_name(KEEP_OPTION)
+                .value_name("KEEP")
+                .short("k")
+                .long(KEEP_OPTION)
+                .multiple(true)
+                .number_of_values(1)
+                .required(false)
+                .help("Regular expression of repository names to keep despite space constraints"),
```

Comment:
Can we use the same verb tense/usage as the other flags? So maybe something like:

```
Prevents Docuum from deleting images from repositories matching <REGEX>
```